gp-advanced-select/gpadvs-enable-add-new-option.php

```diff
@@ -37,6 +37,7 @@ public function init() {
 
 		add_filter( 'gform_pre_render', array( $this, 'allow_created_choices_in_save_and_continue' ), 10, 3 );
 
+		add_filter( 'gform_validation', array( $this, 'validate' ) );
```

Comment:
Gravity Forms is strongly favoring the [gform_field_validation](https://docs.gravityforms.com/gform_field_validation/) filter these days. Let's roll with that one. It will simplify the `validate()` function as well.

Reply:
@spivurno Oh.. Okay. Will make the changes. Thanks for pointing that out :)

Reply:
This probably has to do with state validation. You can prevent state validation by setting `validateState` on the field to `false` during `gform_pre_render`.

We already do this some in GP Advanced Select when lazy-loading with GPPA: https://github.com/gravitywiz/gp-advanced-select/blob/34dc8d067dde0d5797979e004f97c9ba2cbc3fb7/class-gp-advanced-select.php#L388-L396